test/org/sagebionetworks/bridge/validators/ScheduleValidatorTest.java

```diff
@@ -31,6 +31,21 @@ public void before() {
     }
     
     @Test
+    public void oneTimeScheduleWithExpirationNotUnderOneDay() {
+        schedule.setScheduleType(ScheduleType.ONCE);
+        schedule.setExpires(Period.parse("PT23H"));
+        schedule.addActivity(TestConstants.TEST_3_ACTIVITY);
+        try {
+            Validate.entityThrowingException(validator, schedule);
+            fail("Should have thrown InvalidEntityException");
+        } catch(InvalidEntityException e) {
+            assertEquals(
+                    "scheduleType set to once with no time of day start, but expires in under a day (this can cause confusing behavior)",
+                    e.getErrors().get("scheduleType").get(0));
+        }
+    }
```

Comment:
Would it be worthwhile to have two tests here, one with an expires < 1 day, one with expires > 1 day, so we can verify both cases?